lib/test.js

```diff
@@ -52,6 +52,7 @@ const test = (suite, buildConfig = config.defaultBuildConfig, options) => {
   } else {
     util.run('ninja', ['-C', config.outputDir, "fix_brave_test_install_name"], config.defaultOptions)
     util.run('ninja', ['-C', config.outputDir, "fix_brave_test_install_name_adblock"], config.defaultOptions)
+    util.run('ninja', ['-C', config.outputDir, "fix_brave_test_install_name_speedreader"], config.defaultOptions)
```

Comment:
this should have gone in https://github.com/brave/brave-browser/blob/ie_speedreader/lib/test.js